deploy/deploy.go

```diff
@@ -510,7 +510,17 @@ func (k *KindSpec) setupGoogleArtifactRegistryAccess() error {
 	if err := execer.Exec("gcloud", "auth", "print-access-token"); err != nil {
 		return err
 	}
-	execer.SetStdout(log.StandardLogger().Out)
+	// It would be nice to something like:
+	//
+	//	errlog := logshim.New(func(args ...interface{}) {
+	//		log.InfoDepth(2, args...)
+	//	})
+	// So all the log lines don't just say they are coming from logshim, but
+	// it really doesn't help because the lines are coming from a goroutine
+	// that is using io.Copy.  So we just let the logs say logshim.
```

Comment:
nit: maybe clean up this comment, not sure how much clarity it adds in its current state